addons/website_sale/models/product_misc.py

```diff
@@ -208,5 +210,7 @@ def _search_get_detail(self, website, order, options):
     def _search_render_results(self, fetch_fields, mapping, icon, limit):
         results_data = super()._search_render_results(fetch_fields, mapping, icon, limit)
         for data in results_data:
+            if data['website_description']:
+                data['website_description'] = Markup(html2plaintext(str(data['website_description'])))
```

Comment:
What about marking the `description` field in `_search_get_detail()` as being `html` ?
```py
            mapping['description'] = {'name': 'website_description', 'type': 'text', 'html': True, 'match': True}
```
It then gets automatically handled by `_search_render_results()`'s `super`.